src/recyclerlistview/RecyclerListView.js

```diff
@@ -94,6 +97,9 @@ class RecyclerListView extends Component {
 
 
     componentDidUpdate() {
+        if(!this._initComplete){
```

Comment:
Same new props are being used, we need to fix how virtual renderer is initialized